lib/map_view_type.dart

```diff
@@ -0,0 +1,8 @@
+
+class MapViewType {
+  static const int MAP_TYPE_HYBRID = 4;
```

Comment:
Should use dart style here: 
none,
hybrid,
satellite,
terrain.

Also, are you confident these int values map the same on iOS and Android?